base_changeset/models/base.py

```diff
@@ -114,7 +114,7 @@ def write(self, values):
         for record in self:
             local_values = self.env["record.changeset"].add_changeset(record, values)
             super(Base, record).write(local_values)
-        return self
+        return bool(self)
```

Comment:
Good catch! I would simply return True as does the upstream write method: https://github.com/odoo/odoo/blob/12.0/odoo/models.py#L3403

Reply:
Also, please remove internal ticket number and add the module name in the commit message `[FIX] base_changeset: fix write return type`.